modules/tracker/models/pdo/SubmissionModel.php

```diff
@@ -223,4 +308,38 @@ public function getTrends($submissionDao, $key = true)
 
         return $trendDaos;
     }
+
+    /**
+     * Return all distinct branch names of revisions producing submissions.
```

Comment:
This comment bothers me, probably because I don't understand what a revision is.  This is the only mention of 'revision' in this file, and the method returns all distinct branch names from any submission.

I'd prefer

    Return all distinct branch names from submissions.

or to understand revisions better.